tests/testthat/test-coercion.R

```diff
@@ -90,3 +90,129 @@ test_that("as_epidist works for ebola serial interval (issue #303)", {
   # populate mean and sd summary statistics without uncertainty
   expect_true(all(!is.na(ebola_serial_epidist$summary_stats[c("mean", "sd")])))
 })
+
+test_that("as_epidist works for ebola SI assumed prob_dist (issue #310)", {
+  # {epireview} is not a dependency so only run if already on system
+  skip_if_not_installed("epireview")
```

Comment:
Thinking again about this, it may be good to add epireview in `Config/Needs/check` in `DESCRIPTION` to ensure the test runs on CI as well.

Reply:
Resolved in [acac152](https://github.com/epiverse-trace/epiparameter/pull/335/commits/acac1525ebcb59b2c5b91159070cc472e756e9d5).